HLTrigger/Configuration/python/Phase2/ESProducersSources/Chi2MeasurementEstimator_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+Chi2MeasurementEstimator = cms.ESProducer("Chi2MeasurementEstimatorESProducer",
```

Comment:
I note that in confdb configs we have multiple hltChi2\*  variants.
If this is intended to be hlt-specific, why not make clones (it would need less changes later)?

Reply:
That's what we're doing, right now (and in the next commit), it will look this:
```
import FWCore.ParameterSet.Config as cms

from TrackingTools.KalmanUpdators.Chi2MeasurementEstimatorDefault_cfi import Chi2MeasurementEstimatorDefault as _Chi2MeasurementEstimatorDefault

Chi2MeasurementEstimator = _Chi2MeasurementEstimatorDefault.clone()
```

Reply:
This was addressed!